snapcraft.yaml

```diff
@@ -27,9 +29,18 @@ parts:
       - lib
       - sbin
     stage-packages:
+      - coreutils
+      - dosfstools
       - e2fsprogs
       - fakeroot
       - mtools
       - python3-debian
       - python3-parted
       - util-linux
+    install: |
+      TRIPLET="$(uname -p)-linux-gnu"
+      mv $SNAPCRAFT_PART_INSTALL/usr/lib/$TRIPLET/* $SNAPCRAFT_PART_INSTALL/usr/lib/
```

Comment:
Overall, I find this more elegant than the wrapper script.  However, note that you've dropped the logic from the wrapper which handles the cases where uname -p != GNU triplet.  (Also, this code is not safe in the face of cross-building of snaps since the source directory name will not match the current architecture name.)

I recommend simply using globbing here - instead of defining $TRIPLET you should be able to do $SNAPCRAFT_PART_INSTALL/usr/lib/*-linux-gnu*/*.  This would only ever fail if you had libraries for more than one architecture staged and these libraries had file collisions, which is not a concern at this point in time.